tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,234 @@
-
-raw_delta = (1 / raw.info['sfreq'])
-print('raw.first_sample is {}'.format(raw.first_samp * raw_delta))
-print('annot_none.onset[0] is {}'.format(annot_none.onset[0]))
-print('raw_a.annotations.onset[0] is {}'.format(raw_a.annotations.onset[0]))
-
-###############################################################################
-# Valid operations in :class:`mne.Annotations`
-# --------------------------------------------
+# The Events and Annotations data structures
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Concatenate
-# ~~~~~~~~~~~
+# Generally speaking, both the Events and :class:`~mne.Annotations` data
+# structures serve the same purpose: they provide a mapping between times
+# during an EEG/MEG recording and a description of what happened at those
+# times. In other words, they associate a *when* with a *what*. The main
+# differences are:
 #
-# It is possible to concatenate two annotations with the + operator (just like
-# lists) if both share the same ``orig_time``
-
-annot = mne.Annotations(onset=[10], duration=[0.5],
-                        description=['foobar'],
-                        orig_time=orig_time)
-annot = annot_orig + annot  # concatenation
-print(annot)
-
-###############################################################################
-# Iterating, Indexing and Slicing :class:`mne.Annotations`
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# 1. the Events data structure represents the *when* in terms of samples,
+#    whereas the :class:`~mne.Annotations` data structure represents the *when*
+#    in seconds.
+# 2. the Events data structure represents the *what* as an integer "Event ID"
+#    code, whereas the :class:`~mne.Annotations` data structure represents the
+#    *what* as a string.
+# 3. Events in an Event array do not have a duration (though it is possible to
+#    represent duration with pairs of onset/offset events within an Events
+#    array), whereas each element of an :class:`~mne.Annotations` object
+#    necessarily includes a duration (though the duration can be zero if an
+#    instantaneous event is desired).
+# 4. Events are stored as an ordinary :class:`NumPy array <numpy.ndarray>`,
+#    whereas :class:`~mne.Annotations` is a :class:`list`-like class defined in
+#    MNE-Python.
 #
-# :class:`~mne.Annotations` supports iterating, indexing and slicing.
-# Iterating over :class:`~mne.Annotations` and indexing with an integer returns
-# a dictionary. While slicing returns a new :class:`~mne.Annotations` instance.
 #
-# See the following examples and usages:
-
-# difference between indexing and slicing a single element
-print(annot[0])  # indexing
-print(annot[:1])  # slicing
-
-###############################################################################
-# How about iterations?
-
-for key, val in annot[0].items():  # iterate on one element which is dictionary
-    print(key, val)
+# What is a STIM channel?
```

Comment:
you go into a lot of pain to explain what is a STIM channel but there is no explanation what it stands for. Reading it as it is, it seems that it is an abbreviation which not expanded. But in reality it is just the short for "stimulation", right?

Reply:
Personally I always thought of it as the "stimulus" channel not the "stimulation" channel, but I don't know the history of the terminology.  But the first mention at the end of this paragraph includes a glossary link. Is that not enough?

Reply:
yeah I noticed that after reading further. I don't know either -- perhaps stimulus channel is how it is. The mention at the end of the paragraph seems a bit too discrete to me. I would say upfront that it stands for stimulus.

An anecdote that comes to mind is how after many months of working with MNE, @sappelhoff [realized that meas date stands for "measurement date"](https://github.com/mne-tools/mne-bids/pull/53#discussion_r202525422). What may be obvious to us may not obvious to others!

Reply:
also perhaps make it lower case with a period or full stop at the end as it's a shortened form, not an abbreviation